esy/Manifest.ml

```diff
@@ -299,7 +299,7 @@ end = struct
   end
 
   type manifest = {
-    name : string;
+    name : (string [@default "root"]);
```

Comment:
Could we make it `string option` instead? In the case of `None` we could just use basename of package's dir as a package name, e.g. — package defined in `node_modules/hello/package.json` would have the name `hello`, if `package.json` `"name"` is not defined.

Reply:
@andreypopp Where would be the best place to apply the default case instead?